common-libs/did-auth-lib/src/DidAuthService/DidAuthService.ts

```diff
@@ -118,15 +119,15 @@ export default class AffinidiDidAuthService {
   ): Promise<string> {
     const cloudWallet = new CloudWalletApiService({
       accessApiKey: apiKey,
-      cloudWalletUrl: `https://cloud-wallet-api.${environment}.affinity-project.org/api/v1/utilities/sign-jwt`,
+      cloudWalletUrl: resolveUrl(Service.CLOUD_WALLET_API, <Env>environment) + '/api/v1/utilities/sign-jwt',
```

Comment:
It would be better if instead `createDidAuthResponseTokenThroughCloudWallet` just accepted `environment: Env`.